packages/MarkdownEditor-Core.package/MarkdownBlockParser.class/instance/parse..st

```diff
@@ -0,0 +1,13 @@
+parsing
+parse: aString
+
+	| styledText |
+	styledText := aString asText.
+	aString lineIndicesDo: [:start :endWithoutDelimiters :end | | line |
+		line := aString copyFrom: start to: end.
+		(MarkdownHeading matches: line)
+			ifTrue: [
+				(MarkdownHeading attributesFor: line) do: [:attribute |
```

Comment:
Why line break before?

Reply:
The method isn't settled yet, so I think formatting will come later